apps/frontend/src/app/components/footer/footer.component.html

```diff
@@ -17,7 +17,22 @@ <h4>Linkek</h4>
       <li><a href="https://verseghy-gimnazium.net/">Vissza a régi oldalra</a></li>
     </ul>
   </div>
-  <div class="footerblock subscribe">
+  <div class="footerblock ekreta">
+    <h4>KRÉTA</h4>
+    <ul>
+      <li>
+        <a href="https://klik035992001.e-kreta.hu/Adminisztracio/Login" target="_blank">
+          <img alt="" src="assets/icons/kreta2.png" />
+        </a>
+      </li>
+      <li>
+        <a href="https://eugyintezes.e-kreta.hu/" target="_blank">
+          <img alt="" src="assets/icons/eugy.png" width="120" />
+        </a>
+      </li>
+    </ul>
+  </div>
+  <!-- div class="footerblock subscribe">
```

Comment:
When we comment HTML we leave the code to be commented intact and just add the comment tags like in any other language

Reply:
Fixed in 2239f82